apps/android/LiveVideoApp/app/src/main/java/com/twilio/livevideo/app/manager/sync/SyncMapWrapper.kt

```diff
@@ -0,0 +1,102 @@
+package com.twilio.livevideo.app.manager.sync
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.twilio.sync.ErrorInfo
+import com.twilio.sync.EventContext
+import com.twilio.sync.SuccessListener
+import com.twilio.sync.SyncClient
+import com.twilio.sync.SyncMap
+import com.twilio.sync.SyncMapObserver
+import com.twilio.sync.SyncMapPaginator
+import com.twilio.sync.SyncOptions
+import org.json.JSONObject
+
+class SyncMapWrapper constructor(private var uniqueName: String = "") : SyncObject {
```

Comment:
The name of this is less of an issue than the document class, but I think the same logic applies. The name currently implies that this class is to be used with any sync object that is a map, which isn't really true. This class is really only to be used for sync maps that have this specific structure where the identifier is the user identity. We could add other maps in the future that are used completely differently.

This is minor stuff, but I think useful to have a less generic name.

Reply:
Done!